doc/conf.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Imports
+
+import sphinx_rtd_theme
+
+# -- Project information -----------------------------------------------------
+
+project = 'galgebra'
+copyright = '2018, Alan Bromborsky and Galgebra team'
+author = 'Alan Bromborsky'
+
+# The short X.Y version
+version = ''
+# The full version, including alpha/beta/rc tags
+release = '0.4.1.1' 
```

Comment:
Should use a constant from setup.py or somewhere common. I see it's also redundant in clifford.

Reply:
Yeah this is annoying for both libraries, maybe we can come up with a nice solution, probably not a big problem unless we start doing a lot of releases :)

Reply:
Yeah, it's not a big deal. We can just keep in mind to update both of them for every release for the time being.

Reply:
https://github.com/computationalmodelling/nbval/blob/master/nbval/_version.py has a good solution for this.